src/main/webapp/js/project_repo_search.js

```diff
@@ -0,0 +1,35 @@
+Behaviour.specify(".searchable", 'searchableField', 200, function(el) {
+    var results = {};
+
+    new ComboBox(el, function(query) {
+        return Object.keys(results);
+    }, {});
+
+    el.addEventListener('input', function(e) {
+        var parameters = (el.getAttribute("fillDependsOn") || "").split(" ")
+            .reduce(function(params, fieldName) {
+                var dependentField = findNearBy(el, fieldName);
```

Comment:
Yes. It can be found [here](https://github.com/jenkinsci/jenkins/blob/86e17f90f6d9d8c5495c55278648d2f89d45ce04/war/src/main/webapp/scripts/hudson-behavior.js#L192-L242)